packages/tendermint-rpc/src/adaptorforversion.ts

```diff
@@ -4,14 +4,18 @@
 import { Adaptor } from "./adaptor";
 import { v0_33 } from "./v0-33";
 
+const hashes = {
+  v0_34: "ca2c9df",
```

Comment:
The commit hash from the latest tag: https://github.com/tendermint/tendermint/tree/v0.34.0-rc6.

If v0_34 can be used for a bunch of hashes, I think we need a list of hashes here.

```suggestion
  v0_34: ["ca2c9df"],
```